src/bmeg/vertex.py

```diff
@@ -28,21 +28,17 @@ def from_dict(cls, data):
 class Callset(Vertex):
     tumor_aliquot_id: str
     normal_aliquot_id: Union[None, str]
-    call_method: str
```

Comment:
why is this being removed? I think we need a way to distinguish call sets on the same set of aliquots that were analyzed using different methods.

Reply:
Talked to Kyle, sounds like we will keep `source` on `Callset` and put `call_methods` as an array of strings on the `AlleleCall` edges.